Utils/python/pdfweightproducer_cfi.py

```diff
@@ -8,6 +8,6 @@
 )
 
 from TreeMaker.TreeMaker.TMEras import TMeras
-(TMeras.TM2017 | TMeras.TM2018).toModify(PDFWeightProducer,
+(TMeras.TMUL2017 | TMeras.TMUL2018).toModify(PDFWeightProducer,
```

Comment:
this modification probably isn't needed anymore - it specifically handled the change in PDFs for pre-legacy MC in 2017. We should check the number of replicas for the UL PDFs and make sure the right number is used for all years.

Reply:
Yeah, I dumped the headers for a sample across the years and now see that things are the same. So I updated in d01ea7d to simplify things down.